src/runtime_src/core/pcie/driver/linux/xocl/subdev/mb_scheduler.c

```diff
@@ -4940,15 +4967,25 @@ kds_custat_show(struct device *dev, struct device_attribute *attr, char *buf)
 		sz += sprintf(tembuf+sz, ",%d", ert_cq_slot_status(xert, idx));
 	}
 	sz += sprintf(tembuf+sz, "}\n");
+	if (sz >= PAGE_SIZE) {
+		sz = sz_tmp;
+		truncated = true;
+		goto out;
+	}
 
 out:
-	if (sz)
+	tembuf[sz++] = 0;
+	if (sz >= (ssize_t)PAGE_SIZE) {
+		sz = PAGE_SIZE-50; //save truncated CU status
```

Comment:
1. what is the magic number 50 for?
2. please use /* */ instead of //
3. better have a space like sz = PAGE_SIZE - 50

Reply:
Updated. 50 is just some space to add "TRUNCATED" to the end. 50 changed to 30.